site/sass/components/_item-grid.scss

```diff
@@ -130,7 +130,7 @@
       margin-top: 1.5rem;
 
       @include mq(1058px) {
-        grid-template-columns: repeat(auto-fit, minmax(pxRem(400px), 1fr));
+        grid-template-columns: repeat(auto-fill, minmax(pxRem(400px), 1fr));
```

Comment:
Why did you swap this over to fill?

Reply:
Maybe make this something just for the stories modifier for item grid then. I think we in general want `auto-fit` and swapping it here may have unintended consequences elsewhere

Reply:
yup, that's what I did, look ⤵️ 

<img width="633" alt="image" src="https://user-images.githubusercontent.com/1170192/159821908-69a3b9a0-f966-407d-9592-30f9c52ca7b4.png">

Reply:
Oh I totally missed this! Couldn't tell from how the PR diff worked!